src/server/dfly_main.cc

```diff
@@ -594,7 +595,20 @@ void sigill_hdlr(int signo) {
   exit(1);
 }
 
+void print_basic_usage_info() {
+  std::cout << "* Default logdir is: \"/tmp/\". It can be overwritten with flag --log_dir=[path]\n";
+  std::cout << "* Logfile name format is: ";
+  std::cout << "dragonfly.[machine_name].[user_name].[WARNING|INFO|ERROR].[date and time]\n";
+  std::cout << "* For usage info flags type dragonfly [--help | --helpfull]\n";
+  const string df_website = "https://www.dragonflydb.io/";
+  std::cout << "* You can read more about dragonflydb at: " << df_website << '\n';
+  const string df_docs = "https://www.dragonflydb.io/docs";
+  std::cout << "* Documentation can be found at: " << df_docs;
+  std::cout << endl;
+}
+
 int main(int argc, char* argv[]) {
+  print_basic_usage_info();
```

Comment:
I think this call should be after `MainInitGuard`, otherwise, all the absl/logging stuff has not been properly initialized.

Reply:
In addition, the case for the function is not Google styleguide :)